common/state_edicts/02_promote_military_recruitment.txt

```diff
@@ -36,100 +36,121 @@ edict_promote_military_recruitment = {
 	}
 	
 	modifier = {
-		local_manpower_modifier = 0.50
-		local_manpower = 0.250
+		local_manpower_modifier = 0.33
+		#local_manpower = 0.250
 	}
 	
 	color = {  188 90 27  }
 	
 	ai_will_do = {
-		factor = 100
-		
-		# increase interest for each 10% below 50%
-		modifier = {
-			factor = 1.25
-			NOT = {
-				owner = {
-					manpower_percentage = 0.5
-				}
-			}
-		}
-		modifier = {
-			factor = 1.25
-			NOT = {
-				owner = {
-					manpower_percentage = 0.4
-				}
-			}
-		}
+		factor = 5
```

Comment:
The AI is still picking the edict with all of the modifier changes?

Reply:
AI will use it if there is a province with barracks of any tier, or a province with high base manpower.